core/dbt/events/types.py

```diff
@@ -1785,28 +1792,38 @@ def message(self) -> str:
 
 @dataclass
 class PrintStartLine(InfoLevel, Cli, File):
+    # TODO: add node info
+    # report_node_data: ParsedModelNode
     description: str
     index: int
     total: int
+    status: str = 'RUN'
     code: str = "Z031"
 
     def message(self) -> str:
         msg = f"START {self.description}"
-        return format_fancy_output_line(msg=msg, status='RUN', index=self.index, total=self.total)
+        return format_fancy_output_line(
+            msg=msg,
+            status=self.status,
+            index=self.index,
+            total=self.total
+        )
 
 
 @dataclass
 class PrintHookStartLine(InfoLevel, Cli, File):
+    # report_node_data: CompiledModelNode
     statement: str
     index: int
     total: int
     truncate: bool
+    status: str = 'RUN'
     code: str = "Z032"
 
     def message(self) -> str:
         msg = f"START hook: {self.statement}"
         return format_fancy_output_line(msg=msg,
-                                        status='RUN',
+                                        status=self.status,
```

Comment:
Definitely an improvement to structure it like this. It is a bit odd that we call this `RUN` instead of something like `START`, but for this specific event type, it's really just a question of CLI cosmetics